serverless.yml

```diff
@@ -41,3 +41,9 @@ functions:
       - http:
           path: healthcheck
           method: get
+  moviesByGenre:
```

Comment:
There's a section in this file on L36 called `environment`. You need to include env variables here to make them work when we deploy.

Reply:
Good catch!